Resources/SteamLanguage/enums.steamd

```diff
@@ -1205,3 +1205,36 @@ enum EUCMFilePrivacyState flags
 
 	All = Public | FriendsOnly | Private; // 14
 };
+
+enum ELobbyType
+{
+    Private = 0;
+    FriendsOnly = 1;
+    Public = 2;
+    Invisible = 3;
+};
+
+enum ELobbyFilterType
+{
+    String = 0;
+    Numerical = 1;
+    Distance = 4;
+};
+
+enum ELobbyFilterComparison

```

Comment:
SDK name is `ELobbyComparison`

Reply:
I'm _am_ open to changing this. However, what's the precedent here? SteamKit is not Steamworks.

Personally, I think `ELobbyComparison` is a pretty poorly "scoped" name; however, honestly I'm not that attached to it.

If the precedent is to copy Steamworks, then I'll make the change.

Reply:
SteamKit tries to follow Valve's names (SDK or RE). Makes it easier to cross-reference and update.

Reply:
Alright, I'll make the change.

However, is that policy documented somewhere? If not, would be good to jot it down so future contributors know.

Reply:
There's not much that's documented full stop